src/modules/showReservation.js

```diff
@@ -0,0 +1,62 @@
+const showReservation = (obj) => {
+  const {
+    short_title: shortTitle,
+    performers: [{ image: performerImage }],
+    type: tipo,
+    datetime_local: dateEvent,
+    venue: { city: ciudad, country: pais, state: estado },
+    stats: { average_price: precio },
+  } = obj;
+
+  const div = document.createElement('div');
+  div.className = 'popup';
+  div.setAttribute('id', 'popup1');
+  div.innerHTML = `
+                <div class="popup-wrap">
+                <h2>Reserve Now</h2>
+                <a href="#" class="close" onclick="this.parentElement.parentElement.remove()" >&times;</a>
+                <div class="content">
+                <div class="container-f">
+                    <div class="img-thumb">
+                    <img src="${performerImage}" alt="Event Image">
+                    </div>
+                    <div class="details">
+                    <h2>${shortTitle}</h2>
+                    <div class="e-details">
+                        <p>Type: ${tipo}</p>
+                        <p>Date: ${dateEvent}</p>
+                    </div>
+                    <div class="e-details">
+                        <p>Address: ${pais}, ${estado}, ${ciudad}</p>
+                        <p>Price: ${precio}$</p>
```

Comment:
Price field right now shows `$null`. Also you might want to use this fields from API for better UX:

- performers[0].name
- address
- city
- capacity
- datetime_utc